extensions/common/vault/vault-hashicorp/src/test/java/org/eclipse/edc/vault/hashicorp/PathUtilTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright (c) 2022 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.edc.vault.hashicorp;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class PathUtilTest {
+
+    private static Stream<Arguments> provideStringsForTrimsPathsCorrect() {
```

Comment:
I know that I'm getting pedantic about a nit, but I thing using the `ArgumentSource` trough a class extending `ArgumentProvider` instead of the `MethodSource` is way better.

Reply:
I agree with you, it simply slipped through during the copy-paste marathon :)